miden-tx/src/testing/executor.rs

```diff
@@ -37,13 +37,7 @@ impl<H: Host> CodeExecutor<H> {
     }
 
     /// Compiles and runs the desired code in the host and returns the [Process] state
-    ///
-    /// If a module file path was set, its contents will be inserted between `self.imports` and
-    /// `code` before execution.
-    /// Otherwise, `self.imports` and `code` will be concatenated and the result will be executed.
-    pub fn run(self, code: &str) -> Result<Process<H>, ExecutionError> {
-        let assembler = TransactionKernel::assembler().with_debug_mode(true);
-
+    pub fn run(self, code: &str, assembler: Assembler) -> Result<Process<H>, ExecutionError> {
```

Comment:
It seems like this method is always called with the `assembler` parameter set to `TransactionKernel::assembler_testing()`. Should we get rid of the `assembler` parameter just instantiate the assembler as `TransactionKernel::assembler_testing()` in this method?

Reply:
Removed this parameter